lizeth_user.yml

```
@@ -0,0 +1,33 @@
+---
+- name: Setup nepo
+  hosts: devservers
+  become: true
+  environment:
+    HOME: "/home/lizeth"
+    USER: "lizeth"
+  tasks:
+    - name: Create user
+      ansible.builtin.user:
+        name: lizeth
+        groups:
+          - docker
+          - share
+          - sudo
+        shell: "/bin/bash"
+        password: "$6$islasgeci$uMZ4ssQEWitFBBnkTDrBkDV0LSsDK71p0ZyzN0R/vas3rLNpGlyr5O/mb6VtVfOhWy9ufcz/bILMpZ5DvbGwn1"
+    - name: Set authorized keys taken from url
+      ansible.posix.authorized_key:
+        user: lizeth
+        state: present
+        key: https://github.com/LizethMReyes.keys
+    - name: Agrega las credenciales al perfil del usuario
+      ansible.builtin.lineinfile:
+        path: "/home/lizeth/.profile"
+        line: source $HOME/.vault/.secrets
+    - name: Recursively change ownership of home directory
+      ansible.builtin.file:
+        path: "/home/lizeth"
+        state: directory
+        recurse: true
+        owner: lizeth
+        group: lizeth
```

Falta el fin de línea
```suggestion
        group: lizeth
```